numba/cuda/tests/cudadrv/test_context_stack.py

```diff
@@ -74,7 +74,7 @@ class Test3rdPartyContext(SerialMixin, unittest.TestCase):
     def tearDown(self):
         cuda.close()
 
-    def test_attached_primary(self):
+    def test_attached_primary(self, extra_work=lambda: None):
         # Emulate primary context creation by 3rd party
```

Comment:
It doesn't look like any of the callers use this `extra_work` argument.  Is this left over from some experiment?

Reply:
Used in
https://github.com/numba/numba/pull/4354/files#diff-b45f60883d1ad899b1ed781619263591R123-R124

Reply:
oh, argument counting fail on my part!  Looks fine then